webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -80,11 +80,11 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
       label: 'End Time',
     },
     {
-      content: <ul className={css.duration}>
-        <li>Training: {shortEnglishHumannizer(durations.train)}</li>
-        <li>Checkpointing: {shortEnglishHumannizer(durations.checkpoint)}</li>
-        <li>Validation: {shortEnglishHumannizer(durations.validation)}</li>
-      </ul>,
+      content: <div className={css.duration}>
```

Comment:
list items such as `ul`, `ol`, and `li` typically are not consistent across browsers and since we are using `normalize` and not a `reset` we don't get to use them as a blank slate. Opted to use a clean "div" instead.